src/components/common/calendar/Calendar.js

```diff
@@ -1,66 +1,53 @@
 import React, { useState, useEffect } from 'react';
 import '../../../styles/calendar.less';
 import 'antd/dist/antd.css';
-import { Calendar, Modal, Badge, Button } from 'antd';
+import {
+  Calendar,
+  Modal,
+  Badge,
+  Button,
+  Form,
+  Input,
+  DatePicker,
+  TimePicker,
+} from 'antd';
+import moment from 'moment';
 import CalendarModal from './CalendarModal';
-
-const initialValues = [
-  {
-    date: '15/03/2022',
-    type: 'warning',
-    content: 'This is warning event.',
-    details: 'Test information 1',
-    time: '12:00 PM',
-  },
-  {
-    date: '15/03/2022',
-    type: 'success',
-    content: 'This is usual event.',
-    details: 'Test information 2',
-    time: '12:00 PM',
-  },
-  {
-    date: '16/03/2022',
-    type: 'error',
-    content: 'This is error event 1.',
-    details: 'Test information 3',
-    time: '12:00 PM',
-  },
-  {
-    date: '16/03/2022',
-    type: 'error',
-    content: 'This is error event 2.',
-    details: 'Test information 4',
-    time: '12:00 PM',
-  },
-  {
-    date: '16/03/2022',
-    type: 'error',
-    content: 'This is error event 3.',
-    details: 'Test information 5',
-    time: '12:00 PM',
-  },
-  {
-    date: '12/04/2022',
-    type: 'success',
-    content: 'This is usual event1.',
-    details: 'Test information 6',
-    time: '12:00 PM',
-  },
-  {
-    date: '12/04/2022',
-    type: 'success',
-    content: 'This is usual event2.',
-    details: 'Test information 7',
-    time: '12:00 PM',
-  },
-];
+import axiosWithAuth from '../../../utils/axiosWithAuth';
 
 function CalendarApp() {
   const [isModalVisible, setIsModalVisible] = useState(false);
   const [isScheduleModalVisible, setIsScheduleModalVisible] = useState(false);
   const [event, setEvent] = useState(null);
-  const [eventsArr, setEventsArr] = useState(initialValues);
+  const [eventsArr, setEventsArr] = useState([]);
+  const [showEventEditForm, setShowEditEventForm] = useState(false);
```

Comment:
Naming conventions here are reversed! I know it works as is, but you know how much we like naming consistency.